docs/HowToAddAnOperation.md

```diff
@@ -31,11 +31,17 @@ Most operations have constraints on their input and parameters. The best way to
 OpsWithVerifier = ['AveragePool', 'Conv', 'InstanceNormalization', 'Mod']
 ```
 
-The next step will be to invoke the modified `gen_doc.py` file. For this operation, consult the help [here](https://github.com/onnx/onnx-mlir/blob/master/docs/ImportONNXDefs.md)
+The next step will be to invoke the modified `gen_onnx_mlir.py` file. For this operation, consult the help [here](https://github.com/onnx/onnx-mlir/blob/master/docs/ImportONNXDefs.md)
 
 ## Add verifier
 
-You will need to add code in the `src/Dialect/ONNX/ONNXOps.cpp` when the new op was declared as using a verifier.  Best is to look at other operations to get the general pattern, by searching for `static LogicalResult verify(ONNXInstanceNormalizationOp op)`, for example. Note that a verifier will execute each time that one such op is created. So you will need to ensure that it can work with tensors and MemRefs, and possibly unranked tensors. So guard each of your tests to the proper circumstances. For examples, once a tensor is ranked, you may then verify that the rank is within the approved range (if there is such a constraint); before it is ranked, do not perform this test yet.
+You will need to add code in the `src/Dialect/ONNX/ONNXOps.cpp` when the new op was declared as using a verifier.  Best is to look at other operations to get the general pattern, by searching for [static LogicalResult verify(ONNXInstanceNormalizationOp op)](../src/Dialect/ONNX/ONNXOps.cpp#L3395), for example. Note that a verifier will execute each time that one such op is created. So you will need to ensure that it can work with tensors and MemRefs, and possibly unranked tensors. So guard each of your tests to the proper circumstances. For examples, once a tensor is ranked, you may then verify that the rank is within the approved range (if there is such a constraint); before it is ranked, do not perform this test yet.
+
+Tips:
+* Use `operandAdaptor` object to get the inputs and the `op` object to get the attributes.
```

Comment:
This is something I do not quite understand. Is there a reason one cannot retrieve the inputs from the `op`? Why should `operandAdaptor` be used?

Reply:
That is the pattern I have seen being used.

Reply:
To my understanding, the operandAdaptor is constructed with the actual inputs provided to the pattern rewriter. The op contains the  original inputs. Attribute will be the same for both.

Reply:
I noticed too. I am fine with the tip, however getting the input operands from the operator itself *should* also work.

Reply:
Added some text to clarify.